src/pybasic/pybasic.py

```diff
@@ -167,30 +195,20 @@ def fit_predict(self, images: Iterable[ArrayLike]) -> List[np.ndarray]:
             >>> profiles, corrected = basic.fit_predict(images)
         """
         self.fit(images)
-        corrected = self.predict(images)
-        return corrected
+        corrected = self.predict(images, timelapse)
```

Comment:
`return corrected` should be added back or `return self.predict(images, timelapse)`